tests/system/libraries/ChromeLib.py

```diff
@@ -44,7 +44,11 @@ def _getTestCasePath(filename):
 	def start_chrome(self, filePath):
 		builtIn.log(f"starting chrome: {filePath}")
 		self.chromeHandle = process.start_process(
-			f"start chrome \"{filePath}\"",
+			"start chrome"
+			" --force-renderer-accessibility"
+			" --suppress-message-center-popups"
+			" --disable-notifications"

```

Comment:
You could also add --kiosk which will launch Chrome in full-screen, and disable the outer chrome (address bar etc. Which may make the initial tabbing to find the start of the test less complicated.

Reply:
I'm hesitant to do this right now, but but it is worth keeping mind as an option for the future.